extension/main.js

```diff
@@ -29,24 +29,32 @@
 				}
 			}
 
+			if (interval !== 'undefined') {
+				period = 1;
```

Comment:
Hum, it's a big mistake: The test should be ```if (interval === 'undefined')```. It's a dirty test to avoid case of the auth token is empty.